web/src/narrow_error.ts

```diff
@@ -0,0 +1,26 @@
+import render_empty_feed_notice from "../templates/empty_feed_notice.hbs";
+
+interface QueryWord {
```

Comment:
`type QueryWord = {` would be preferred to match the overall trend in our codebase.